src/WorkWithFiles.java

```diff
@@ -0,0 +1,60 @@
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.nio.ByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.util.Scanner;
+
+class WorkWithFiles {
```

Comment:
maybe rename it (something like FileWorker or else)

Reply:
y, its true